<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>603</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Iso-value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="isoValueSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="sliderPosition">
         <number>128</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="horizontalWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="MainopenGLView" name="mainview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>480</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Landmarks</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QWidget" name="verticalWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="locale">
             <locale language="C" country="AnyCountry"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetNoConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListView" name="LandmarkListView">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::ActionsContextMenu</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGraphicsView" name="graphicsView">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>200</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_landmarkset"/>
    <addaction name="action_Open_landmarkset"/>
    <addaction name="action_Save_Landmark_set"/>
    <addaction name="actionSave_landmrk_set_As"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Volume"/>
    <addaction name="separator"/>
    <addaction name="action_Create_template"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Landmark">
    <property name="title">
     <string>&amp;Landmark</string>
    </property>
    <addaction name="action_Add"/>
    <addaction name="action_Set"/>
    <addaction name="action_Edit"/>
    <addaction name="action_Delete"/>
    <addaction name="action_Clear"/>
    <addaction name="separator"/>
    <addaction name="action_Clear_all_locations"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Anterior"/>
    <addaction name="action_Posterior"/>
    <addaction name="action_Left"/>
    <addaction name="action_Right"/>
    <addaction name="action_Head_first"/>
    <addaction name="action_eet_first"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuFoto">
    <property name="title">
     <string>&amp;Foto</string>
    </property>
    <addaction name="action_This_view"/>
    <addaction name="action_All_landmarks"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Landmark"/>
   <addaction name="menuFoto"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Volume"/>
   <addaction name="separator"/>
   <addaction name="action_Open_landmarkset"/>
   <addaction name="action_Save_Landmark_set"/>
   <addaction name="actionSave_landmrk_set_As"/>
   <addaction name="separator"/>
   <addaction name="action_This_view"/>
   <addaction name="action_All_landmarks"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Volume">
   <property name="icon">
    <iconset resource="lmpick.qrc">
     <normaloff>:/icons/icons/document-open-volume.png</normaloff>:/icons/icons/document-open-volume.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;Volume</string>
   </property>
  </action>
  <action name="action_Open_landmarkset">
   <property name="icon">
    <iconset resource="lmpick.qrc">
     <normaloff>:/icons/icons/document-open.png</normaloff>:/icons/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open landmark set</string>
   </property>
  </action>
  <action name="action_Save_Landmark_set">
   <property name="icon">
    <iconset resource="lmpick.qrc">
     <normaloff>:/icons/icons/document-save.png</normaloff>:/icons/icons/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Landmark set</string>
   </property>
  </action>
  <action name="actionSave_landmrk_set_As">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/document-save-as.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save landmrk set &amp;As </string>
   </property>
  </action>
  <action name="action_Create_template">
   <property name="text">
    <string>&amp;Create template</string>
   </property>
  </action>
  <action name="action_New_landmarkset">
   <property name="text">
    <string>&amp;New landmarkset</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Add">
   <property name="text">
    <string>&amp;Add Landmark</string>
   </property>
   <property name="iconText">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add new landmark </string>
   </property>
  </action>
  <action name="action_Edit">
   <property name="text">
    <string>&amp;Edit Landmark</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>&amp;Delete Landmark</string>
   </property>
  </action>
  <action name="action_Clear">
   <property name="text">
    <string>Clear &amp;Location</string>
   </property>
  </action>
  <action name="action_Clear_all_locations">
   <property name="text">
    <string>&amp;Clear all locations</string>
   </property>
  </action>
  <action name="action_Anterior">
   <property name="text">
    <string>&amp;Anterior</string>
   </property>
  </action>
  <action name="action_Posterior">
   <property name="text">
    <string>&amp;Posterior</string>
   </property>
  </action>
  <action name="action_Left">
   <property name="text">
    <string>&amp;Left</string>
   </property>
  </action>
  <action name="action_Right">
   <property name="text">
    <string>&amp;Right</string>
   </property>
  </action>
  <action name="action_Head_first">
   <property name="text">
    <string>&amp;Head first</string>
   </property>
  </action>
  <action name="action_eet_first">
   <property name="text">
    <string>&amp;Feet first</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_This_view">
   <property name="icon">
    <iconset resource="lmpick.qrc">
     <normaloff>:/icons/icons/snapshot.png</normaloff>
     <normalon>:/icons/icons/auto_snapshot_on.png</normalon>:/icons/icons/snapshot.png</iconset>
   </property>
   <property name="text">
    <string>&amp;This view</string>
   </property>
  </action>
  <action name="action_All_landmarks">
   <property name="icon">
    <iconset resource="lmpick.qrc">
     <normaloff>:/icons/icons/auto_snapshot_on.png</normaloff>:/icons/icons/auto_snapshot_on.png</iconset>
   </property>
   <property name="text">
    <string>&amp;All landmarks</string>
   </property>
  </action>
  <action name="action_Set">
   <property name="text">
    <string>&amp;Set Landmark</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainopenGLView</class>
   <extends>QOpenGLWidget</extends>
   <header>mainopenglview.hh</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="lmpick.qrc"/>
 </resources>
 <connections/>
</ui>
